### ISSUE TITLE
How can I enable this plugin's schema.json?

### ISSUE THREAD

**Issue by dbarnett** [mattn/vim-lsp-settings#372]
Is there a way to configure a JSON schema for my vim-lsp-settings settings.json files? When I do e.g. `:LspSettingsGlobalEdit` and edit my JSON configs, it would be so helpful to have completions and inline help for the different config options.

**Comment by dbarnett** [comment]
I was getting ready to draft up a JSON schema to send to schemastore and noticed there's a schema.json and local-schema.json in this repo. Is that for the same JSON settings structure, and is there a way to hook that up to be used for my settings.json files?

**Comment by yaegassy** [comment]
@dbarnett Do you have `json-languageserver` installed?

```
:LspInstallServer json-languageserver
```

It is available when json-languageserver is installed.

<img width="1050" alt="vim-lsp-settings-schema" src="https://user-images.githubusercontent.com/188642/104288769-95b31c80-54fb-11eb-94e7-dbbe7461a833.png">

**Comment by dbarnett** [comment]
Yep, this is with json-languageserver installed. I couldn't get it to show any smart suggestions or anything on settings.json, and it does for other JSON configs.

How is the detection supposed to work for those? Is there a custom fileMatch rule built into the plugin, or somewhere in the LS or schemastore already?
